src/jwplayer/utils/scriptLoader.ts

```diff
@@ -0,0 +1,54 @@
+class ScriptLoader {
+	createScript(
+		src: string,
+		type = 'text/javascript',
+		isAsync = true,
+		node: HTMLElement | string = null,
```

Comment:
`null` can't be assigned to the type `string | HTMLElement`. Since you're always passing a string ("first") based on usage, we could just remove the default value, I think.

```suggestion
		node: HTMLElement | string,
```